samples/bluetooth/pbp_public_broadcast_sink/src/main.c

```diff
@@ -427,6 +431,12 @@ int main(void)
 	}
 	printk("Bluetooth initialized\n");
 
+	err = bt_pacs_register(&pacs_param);
```

Comment:
Thanks for catching this :) 

Can I suggest to move the call to `bt_pacs_register` to the `bap_broadcast_sink_init` so it's closer to the call to `bt_pacs_cap_register`?